src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -2040,8 +2041,9 @@ internal bool TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataRead
                 {
                     case TdsEnums.SQLERROR:
                     case TdsEnums.SQLINFO:
+                    case TdsEnums.SQLDEBUG_CMD:
                         {
-                            if (token == TdsEnums.SQLERROR)
+                            if (token == TdsEnums.SQLERROR || token == TdsEnums.SQLDEBUG_CMD) // This the value of 0x60 send fromm TDS Control token
```

Comment:
I don't understand what the comment is trying to communicate here.

Reply:
For the comment since this was a draft PR and I needed to make a change to run the pipelines I did not pay much attention to it.

The specs do not provide any info on that TDS token stream. At first, I did not notice that 0x60 is available in TDSEnums. basically, the issue was raised by PS team. They tried to provide an invalid token(AE token not TDS header token) to decrypt an encrypted column. Server returns 96 instead of error token (0xaa).

Reply:
If we're getting it from the server then either they need to update the spec or stop sending it. I think updating the spec is easier.